# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/quota/QuotaManager.h
# Commit: f466599094b7
# Full Hash: f466599094b7e16948794a49b0571e03eb2b4df9
# Author: Serban Stanca <sstanca@mozilla.com>
# Date: 2025-06-15 13:49:48
# Regressor Bug: 1962133
# File Overlap Count: 14
# Description:
#   Revert "Bug 1962133 - QM: Add accessed flag to FullOriginMetadata and persist it in the origin metadata file; r=dom-storage-reviewers,hsingh" as requested by dmeehan for causing top crash in Bug 1972221.
#   
#   This reverts commit 433768e64dd71c996694e08005854756b7c45fb3.
# ==============================================================================

diff -r 2408bee8bcc4 -r f466599094b7 dom/quota/QuotaManager.h
--- a/dom/quota/QuotaManager.h	Sat Jun 14 05:48:43 2025 +0000
+++ b/dom/quota/QuotaManager.h	Sun Jun 15 16:46:59 2025 +0300
@@ -226,8 +226,6 @@
   void UpdateOriginAccessTime(const OriginMetadata& aOriginMetadata,
                               int64_t aTimestamp);
 
-  void UpdateOriginAccessed(const OriginMetadata& aOriginMetadata);
-
   void RemoveQuota();
 
   void RemoveQuotaForRepository(PersistenceType aPersistenceType) {